### PR TITLE
Bundler runtime watch improvements

### PR DESCRIPTION
**Pull request by dlcousins**
# NOT READY FOR REVIEW

# Summary
- Improvements to the bundling and local development watch operations?

# Review By (Date)
- ASAP

# Criticality
- 8

# Review Tasks

## Setup tasks and/or behavior to test

1. Run this branch locally and check that files are in fact being watched and updated properly. Generally, the issue has been related to server rendered exports and how/where --watch was working.
2. Run `npm run build` first
3. Then run `npm run start`
4. Make edits to any file within a components and save it. You should see something like thia: `components/{component_name}/Component.jsx changed`
5. Refreshing the front end and check that the change has flowed through.

# Associated Issues and/or People
- JIRA ticket(s) - [STANFORD-742](https://squizgroup.atlassian.net/browse/STANFORD-742)
- Anyone who should be notified? @broleomo @iamrentman

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules
-dist
\ No newline at end of file
+dist
+build
+global
```
Comment (dlcousins) [inline .eslintignore:4]: The cause if the pipeline failures. This is where our exported files are placed.

File: jest.config.cjs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   collectCoverageFrom: ["./**/*.{js,jsx}"],
-  coveragePathIgnorePatterns: ["node_modules", "dist", "global/css"],
+  coveragePathIgnorePatterns: ["node_modules", "dist", "global", "build"],
```
Comment (dlcousins) [inline jest.config.cjs:3]: ensures unit test do not run over new exported bundles.

**Comment by iamrentman** [comment]
I've pulled this branch down and followed the steps under Setup above. I am now able to check changes with just the `start` running instead of also having to run a concurrent `build`. Thanks!

Appreciate the info on the compiled files in each component `dist` as well as looking in to the pipeline failures.

**Review by iamrentman — APPROVED** [review]
I pulled and tested the `build` and `start` – seems to be doing what is expected. Thanks!

**Comment by iamrentman** [comment]
Merging this in. Seems to work as expected and it will stop the pipeline failures that keep happening.

**Comment by dlcousins** [comment]
Thanks @iamrentman 
Very glad to hear it is working a lot better.
Let's discuss also during our next office hours meeting - as it relates to the Automating GitHub global css + JS deploy document  :)

**Pull request merged**